File: scratch/test_whapi.js
const axios = require('axios');
const token = 'shJOb5wskQMTyfoF20GLqmJOclA5if5j';
const target = '120363425790792660@g.us';

axios.post('https://gate.whapi.cloud/messages/text', {
    to: target,
    body: 'TEST FROM NODE VPS SCRIPT (ROBUST)'
}, {
    headers: { 'Authorization': `Bearer ${token}` }
}).then(r => {
    console.log('SUCCESS');
    console.log(JSON.stringify(r.data, null, 2));
}).catch(e => {
    console.error('ERROR');
    console.error(e.response?.data || e.message);
});

File: legacy_scripts/test_whapi.py
import requests

url = "https://gate.whapi.cloud/messages/text"
headers = {
    "Authorization": "Bearer shJOb5wskQMTyfoF20GLqmJOclA5if5j",
    "Accept": "application/json",
    "Content-Type": "application/json"
}
payload = {
    "to": "120363425790792660@g.us",
    "body": "Test message from API"
}

try:
    response = requests.post(url, headers=headers, json=payload)
    print("Status:", response.status_code)
    print("Response:", response.text)
except Exception as e:
    print("Error:", e)
